JOSESwift/Sources/P256Signer.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  P256Signer.swift
+//  JOSESwift
+//
+//  ---------------------------------------------------------------------------
+//  Copyright 2024 Curity AB
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  ---------------------------------------------------------------------------
+//
+
+import CryptoKit
+import Foundation
+
+@available(iOS 13.0, *)
+internal struct P256Signer: SignerProtocol {
+    typealias KeyType = P256.Signing.PrivateKey
+    
```

Comment:
Swiftlint warning: trailing_whitespace

Reply:
swiftlint run and fixed the tool's reported warnings